features/admin/add_buyer_domain.feature

```diff
@@ -0,0 +1,30 @@
+@add-buyer-domain
+Feature: Admin buyer domains
+
+Scenario: Admin user can log in and add a buyer domain
+  Given I am logged in as the production admin user
+  And I am on the /admin page
+  And I click the 'Add a buyer email domain' link
+  When I enter 'test.com' in the 'new_buyer_domain' field
```

Comment:
How do we guarantee that this exists?  It would be good to not rely on pre-existing data set up in our functional tests; that was one of the main reasons for rewriting from scratch last time.

Reply:
We can guarantee gov.uk exists as it is propagated from production :man_shrugging: Changed it to that.